cpp/oneapi/dal/table/backend/csr_table_impl.hpp

```diff
@@ -90,7 +103,17 @@ class csr_table_impl : public detail::csr_table_template<csr_table_impl>,
     }
 
     std::int64_t get_row_count() const override {
```

Comment:
This function is called several times and returns the same every time, maybe set row_count_ one time and leave this function as it was before?

Reply:
I have returned row_count_ back.